frontend/src/stores/levels/state.ts

```diff
@@ -0,0 +1,15 @@
+import type { Level } from '@/domain/Level'
+
+interface LevelState {
```

Comment:
All the interface declarations should be under domain folder. Then you can export it and use it in the actions file